upbound_uxp/assets/monitors/controller_runtime_reconcile_error.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "Upbound UXP {{host.name}} encountered reconciliation errors",
+	"type": "query alert",
+	"query": "sum(last_5m):per_minute(sum:uxp.controller_runtime_reconcile_errors_total{*}.as_count()) > 100",
+	"message": "{{#is_alert}}\n  High reconciliation error rate.\n{{/is_alert}} \n\n{{#is_recovery}}\n  Reconciliation error rate recovered.\n{{/is_recovery}}",
+	"tags": [
+	    "integration:upbound_uxp"
+	],
+	"options": {
+		"thresholds": {
+			"critical": 100,
+			"warning": 80,
+			"critical_recovery": 0,
+			"warning_recovery": 50
+		},
+		"notify_audit": false,
+		"require_full_window": false,
+		"notify_no_data": false,
+		"renotify_interval": 0,
+		"include_tags": false,
+		"escalation_message": "",
+		"new_host_delay": 300,
+		"silenced": {}
+	},
+	"priority": 3,
+	"recommended_monitor_metadata": {
+	    "description": "Get notified when reconciliation errors reach the defined threshold which represents a growing gap in represented managed resource and real resource states."
+	},
+	"restricted_roles": null
+}
```

Comment:
```suggestion
{
	"version": 2,
	"created_at": "2023-09-21",
	"last_updated_at": "2023-09-21",
	"title": "Upbound UXP encountered reconciliation errors",
	"description": "Get notified when reconciliation errors reach the defined threshold which represents a growing gap in represented managed resource and real resource states.",
	"definition": {
		"id": 125257299,
		"name": "Upbound UXP encountered reconciliation errors on {{host.name}}",
		"type": "query alert",
		"query": "sum(last_5m):per_minute(sum:uxp.controller_runtime_reconcile_errors_total{*}.as_count()) > 100",
		"message": "{{#is_alert}}\n  High reconciliation error rate.\n{{/is_alert}} \n\n{{#is_recovery}}\n  Reconciliation error rate recovered.\n{{/is_recovery}}",
		"tags": [],
		"options": {
			"thresholds": {
				"critical": 100,
				"warning": 80,
				"critical_recovery": 0,
				"warning_recovery": 50
			},
			"notify_audit": false,
			"require_full_window": false,
			"notify_no_data": false,
			"renotify_interval": 0,
			"include_tags": false,
			"escalation_message": "",
			"new_host_delay": 300,
			"silenced": {}
		},
		"priority": 3,
		"restricted_roles": null
	},
	"tags": [
		"integration:upbound-uxp"
	]
}
```
Hi, we just released a new version of our recommended monitors schema. This suggestion is to ensure this monitor follows that new format, while keeping all of the content the same.

Reply:
Updated.